vhdbuilder/packer/install-dependencies.sh

```diff
@@ -348,6 +348,11 @@ if grep -q "fullgpu" <<< "$FEATURE_FLAGS" && grep -q "gpudaemon" <<< "$FEATURE_F
 fi
 fi
 
+systemctlEnableAndStart cgroup-memory-telemetry.timer || exit 1
```

Comment:
suggest a dummy `mkdir -p /var/log/azure/Microsoft.Azure.Extensions.CustomScript/events`

to avoid below failing

however, waagent will want to own this dir itself, so immediately remove it